.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+versions:
+  - "2.7"
+  - "3.5"
+cache: pip
+
+install:
+  - pip install -U pip
+  - pip --version
+  - pip install ansible ansible-lint yamllint flake8 molecule docker
+  - ansible --version
+  - ansible-lint --version
+  - yamllint --version
+  - flake8 --version
+  - molecule --version
+
+script:
+  - molecule lint -s openstack
```

Comment:
Why did you remove the `docker` test? I'm guessing there are errors with the service startup in that environment? In that case, let's remove the `molecule/docker` directory as well.